packages/components/molecules/f-skeleton-loader/src/components/skeletons/CarouselCardBackground.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <div class="c-skeleton-loader-carouselCardBackground c-skeleton-loader-bone">
```

Comment:
Can `c-skeleton-loader-bone` be moved to `SkeletonLoader.vue` and added there to `<component :is="skeletonType" />` like `<component :is="skeletonType" class="c-skeleton-loader-bone" />` instead of adding it to every child component?

Reply:
Great point. I have been thinking about ways to improve this passing down or a mixin to generate the class that can be shared. I will address this in the next PRs if that's ok?

Reply:
Another thing I've spotted is that for children components you use for example `c-skeleton-loader-carouselCardBackground` class name. As this class will be used specifically for this component the class name should be just `c-carouselCardBackground` instead